src/shared/js/helpers/blur.js

```diff
@@ -1,4 +1,9 @@
-export const setBlur = ($el, editUrl) => {
+export const setBlur = ($target, editUrl) => {
+  let $el = $target;
+  if (!$el.data('cskEntryId')) {
```

Comment:
shouldn't it be 'csk-entry-id'?, I don't think 'cskEntryId' will trigger anything.

Reply:
Hmm interesting, when accessing with vanilla JS you use `.dataset.cskEntryId` that's where my assumption came from.
And it looks like its working, I'll confirm anyway.